### PR TITLE
Patch "Show QR Code" functionality

### LINKED ISSUE #19878: Patch "Show QR Code" functionality
# Feature Issue

### User Story

As a user
I want to be able to show someone else's qr code
so that i can share their details with someone else

### Description

There's a few places where this feature is not patched through (functionality already implemented):

Home -> Contacts -> Context menu
Group chat -> View members -> context menu

### LINKED ISSUE #19879: Patch "Share profile" functionality
# Feature Issue

### User Story

As a user
I want to be able to share a profile with another user
So that I can connect people in the status network

### Description

There's a few places where the functionality (already implemented), is not triggered:

- Home contacts -> context menu
- Group chats -> View members -> Context menu

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #19878
fixes #19879

just enable existing functionality, not need to manually test

### CODE REVIEW

File: src/status_im/common/home/actions/view.cljs [hunk 1]
```diff
@@ -277,23 +277,29 @@
...
-  []
+  [public-key]
   (entry {:icon                :i/share
           :label               (i18n/label :t/share-profile)
-          :on-press            #(js/alert "TODO: to be implemented")
+          :on-press            (fn []
+                                 (rf/dispatch [:universal-links/generate-profile-url
+                                               {:public-key public-key
+                                                :on-success #(rf/dispatch [:open-share
+                                                                           {:options {:message %}}])}]))
```
Comment (ibrkhalil) [inline src/status_im/common/home/actions/view.cljs:302]: Shouldn't share-profile share the account public key key using `react-native/share `?

**Review by ibrkhalil — APPROVED** [review]
✅

**Comment by yevh-berdnyk** [comment]
hi @flexsurfer, e2e  failures are not PR related, thanks!

**Review by ibrkhalil — APPROVED** [review]

File: src/status_im/common/home/actions/view.cljs [hunk 2]
```diff
@@ -419,9 +425,6 @@
      (notifications-entry false))
    (when (and config/fetch-messages-enabled? inside-chat?)
      (chat-actions/fetch-messages chat-id))
-   (when public?
-     (when config/show-not-implemented-features?
-       (show-qr-entry)))
```
Comment (seanstrom) [inline src/status_im/common/home/actions/view.cljs:424]: Why is this one removed? is it because we don't have the `public-key` in scope?

**Pull request merged**